geode-core/src/main/java/org/apache/geode/distributed/internal/ClusterDistributionManager.java

```diff
@@ -1451,6 +1451,18 @@ public long cacheTimeMillis() {
   }
 
 
+  @Override
+  public DistributedMember getMemberWithName(String name) {
+    for (DistributedMember id : members.values()) {
+      if (id.getName() != null && id.getName().equals(name)) {
```

Comment:
Could `Objects.equals()` be of use here?

Reply:
I second Sarge's suggestion.